documentation/ProjectReference-Protocol.md

```diff
@@ -57,7 +57,8 @@ If implementing a project with an “outer” (determine what properties to pass
   * It returns an item with the following metadata:
     * `TargetFrameworks` indicating what TargetFrameworks are available in the project
     * `TargetFrameworkMonikers` and `TargetPlatformMonikers` indicating what framework / platform the `TargetFrameworks` map to.  This is to support implicitly setting the target platform version (for example inferring that `net5.0-windows` means the same as `net5.0-windows7.0`) as well as treating the `TargetFramework` values [as aliases](https://github.com/NuGet/Home/issues/5154)
-    * Boolean metadata for `HasSingleTargetFramework` and `IsRidAgnostic`.
+    * Boolean metadata for `HasSingleTargetFramework` and `IsRidAgnostic` and `IsVcxOrNativeProj`.
```

Comment:
`SupportsPlatformNegotiation` is a bit too broad for this bool.

`IsVcxOrNativeProj` is used for:

- Should `SetPlatform` be `Platform=` or `PlatformTarget=`?
- Should we work around some Target Framework negotiation logic? (because it's conditioned against .vcxproj and .nativeproj)
- Are we explicitly looking at a .vcxproj or .nativeproj from a managed project? (non .vcxproj/.nativeproj)